### DISCUSSION TITLE
What are your plans to support Mocha 9?

### DISCUSSION THREAD

**Discussion by kberg** [sysgears/mochapack#120] [General]
Thanks for this project. I rely on it every day.

What are your plans to support Mocha v9.x.x? This project's constraints prevent me from upgrading. It's not urgent, but I'd like to move forward, hence why I'm asking what what you're plans are. Thanks for your help.

**Comment by larixer** [comment]
As far as I know, no one is working on Mocha 9 support for this project yet, but pull requests are welcome, of course.

Update. Experimental Mocha 9 support has been added, merged, and published in mochapack@2.1.4:
https://github.com/sysgears/mochapack/pull/124

**Reply by kberg** [reply]
Ack. Once I get done with my long list of other stuff, happy to take a look. But you know, that'll be a while, too. ;)

**Reply by larixer** [reply]
Update. Experimental Mocha 9 support has been added, merged, and published in `mochapack@2.1.4`:
https://github.com/sysgears/mochapack/pull/124

**Comment by kberg** [comment]
Woo hoo!
